debian/kafka-connect-datagen/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Builds a docker image for Kafka Connect
+
+FROM confluentinc/cp-kafka-connect-base
+
+MAINTAINER devx@confluent.io
+LABEL io.confluent.docker=true
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+
+RUN echo "===> Installing Datagen Source Connector ..."
+RUN confluent-hub install confluentinc/kafka-connect-datagen:latest --no-prompt
```

Comment:
are we concerned about compatibility? as this gets maintained on older release branches, `latest` may no longer work with older connect versions (and in general just results in builds that aren't reproducible).

I assume the issue here is that we can't use the same approach re: versions as elsewhere since kafka-connect-datagen is versioned independently? If that's the case, we should probably introduce an `ENV` that sets it explicitly and can be put in place of `latest`.

Reply:
Added ARG and set to current latest version `0.1.2`

Reply:
There is no good reason why kafka-connect-datagen isn't versioned with the rest of CP.  Shoul we perhaps change that first, and then we can use the same approach?

Reply:
¯\_(ツ)_/¯ I don't mind either way :)